1_prep/src/munge_nmls.R

```diff
@@ -0,0 +1,51 @@
+#' @title Make nml edits
+#' @description prepare edits that can be made the same way
+#' for all lakes. Applies to one lake's nml list at at time
+#' @param nml_args lake-specific nml parameters loaded in from
+#' the nml_list.rds from lake-temperature-model-prep
+#' @return a list of modified nml arguments
+make_nml_edits <- function(nml_args) {
+  nml_edits <- with(nml_args, c(
+    # &glm_setup
+    max_layers = max(30, ceiling(7 * lake_depth)),
```

Comment:
I was initially thinking these things perhaps should live in the `prep` pipeline instead, but after considering further, this is a GLM-specific set of changes and I agree it makes sense to be here, since perhaps in the future some other model-specific munge stuff will need to be here for other process-based models.